frontend/src/model/chain.ts

```diff
@@ -45,10 +45,12 @@ export class ChainUtils {
     }
     return undefined;
   }
+
   static getTssKeysignType(chain: Chain): TssKeysignType {
     switch (chain) {
-      case (Chain.Solana, Chain.Polkadot):
+      case "Solana":
```

Comment:
why change it to "Solana" , not use `Chain.Solana`?

Reply:
It was always returning the default;

Reply:
Changed to use the enum.